.gitignore

```diff
@@ -1,5 +1,19 @@
+www/templates_c/
+
 debian/debhelper-build-stamp
 debian/files
 debian/plug-ugmm.debhelper.log
 debian/plug-ugmm.substvars
 debian/plug-ugmm/
+
```

Comment:
This looks like you've added a whole bunch of generic .gitignores to the file. Try and keep the gitignore file specific to the project. (So it's a PHP project, don't have the Python ignores, etc. VIM is used enough, that adding the .swp ignore is ok, although you could argue that it should be in your global git ignore instead)

Reply:
Thanks, done.